.github/workflows/linux.yml

```diff
@@ -68,6 +68,7 @@ jobs:
             echo "::group::CMakeConfig 2"
             cmake -DDD4HEP_HEPMC3_COMPRESSION_SUPPORT=ON ..
           fi
+          # Make sure DD4hep can be built with an older version of EDM4hep (0.10.5 for LCG_106)
           if [[ ${{ matrix.LCG }} =~ 106 ]]; then
```

Comment:
We should also enable this for `104` (as discussed in https://github.com/AIDASoft/DD4hep/pull/1380#discussion_r1906861531)

Reply:
It doesn't work due to the version of `podio` being too old in 104. However I built locally with EDM4hep 0.10.0 and it is possible. I reverted the addition of LCG_104. If it's not possible to easily test against 0.10.0 then 0.10.5 is the best we can do :shrug:

Reply:
LCG_105 has 0.10.2 if we want to try that one

Reply:
In that case we also have to adjust the minimal version of podio in the CMakelists:

https://github.com/AIDASoft/DD4hep/blob/c532581bc637fcedb32e1ecb71b6390328772f3b/CMakeLists.txt#L166

LCG_104 comes with 0.16.6 which is larger than the 0.16.3 we currently require.